File: Python/計實作業/P2-110502557.py
'''
Practice 02
Name: 游宗穎
Student Number: 110502557
Course: 2021-CE1003-B
'''

import random

randnum = random.randint(1, 100)
count = 0
while (True):
    guess = int(input("輸入一數字 : "))
    count += 1
    if (randnum > guess): print("比", guess, "還要大")
    elif (randnum < guess): print("比", guess, "還要小")
    else: break
print("猜對了！總共猜了", count, "次")

File: Java/計實作業/P2_110502557.java
/**
 * Practice 02
 * Student Number: 110502557
 * Course: 2022-CE1004-A
 */

public class P2_110502557 {
    public static void main(String[] args) throws Exception {
        final boolean isWindows = System.getProperty("os.name").toLowerCase().indexOf("win") >= 0;
        final String command = (isWindows ? "copy" : "cp") + " test.txt 110502557.txt";
        Process process = Runtime.getRuntime().exec(command);
        process.waitFor();
    }
}
